apps/explorer/src/ui/SplitPanes.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import clsx from 'clsx';
+import { Fragment, type ReactNode, useState } from 'react';
+import { Panel, PanelGroup, PanelResizeHandle } from 'react-resizable-panels';
+
+export type SplitPaneProps = {
```

Comment:
Could you just make this extend the `PanelGroup` props and then pass them through except for `panels`?

Reply:
done